submissions/adeyemiJuwon/dom/product.js

```diff
@@ -0,0 +1,73 @@
+let quantity = document.getElementById("quantity");
+let price = document.getElementById("price");
+let shippingCost = document.getElementById("shippingCost");
+let totalCost = document.getElementById("totalCost");
+let button = document.getElementById("btn");
+let customer = document.getElementById('customer-price');
+let grossPrice = quantity.value * Number(price.innerHTML);
+
+
+
+function shipp(){
+	grossPrice = quantity.value * Number(price.innerHTML);
+	if ( grossPrice < 1000){
+		shipping = 100
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}
+	else if(grossPrice >= 1000 && grossPrice <= 20000){
+		shipping= 0.15 * grossPrice
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}else if (grossPrice >= 20000 && grossPrice <= 100000){
+		shipping= 0.20 * grossPrice
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}else{
+		shipping = 25000
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}
+	return true
+}
+
+
+
+function cost(){
+
```

Comment:
shipping is not defined in this function @Samju200

Reply:
Okay.
Done